src/transformers/tokenization_utils_base.py

```diff
@@ -2395,6 +2399,16 @@ def _from_pretrained(
         # Instantiate the tokenizer.
         try:
             tokenizer = cls(*init_inputs, **init_kwargs)
+        except DecodeError:
+            warnings.warn(
+                "Unable to load tokenizer model from SPM. Loading from TikToken will be attempted instead.",
+            )
+            return False
+        except RuntimeError:
+            warnings.warn(
+                "Unable to load tokenizer model from SPM. Loading from TikToken will be attempted instead.",
+            )
+            return False
```

Comment:
fine with with me if we can refine the error more than this!

Reply:
@ArthurZucker Okay yes! Refine in what way? The message?

Reply:
update: refined the message slighty